src/routes/docs/components/TreeList/+page.svelte

```diff
@@ -2,8 +2,77 @@
   import TreeList from '$lib/components/TreeList.svelte';
   import Preview from '$lib/components/Preview.svelte';
   import Blockquote from '$docs/Blockquote.svelte';
-</script>
+    import Icon from '$lib/components/Icon.svelte';
+    import { mdiCircleSmall } from '@mdi/js';
+    import { cls } from '$lib';
+
+  type Node = { name: string; level: number; children: Node[] };
 
+
+  let nodes: Node[] = [
+    {
+      name: 'Node 1',
+      level: 1,
+      children: [
+        {
+          name: 'Node 1.1',
+          level: 2,
+          children: [
+            {
+              name: 'Node 1.1.1',
+              level: 3,
+              children: []
+            },
+            {
+              name: 'Node 1.1.2',
+              level: 3,
+              children: []
+            }
+          ]
+        },
+        {
+          name: 'Node 1.2',
+          level: 2,
+          children: []
+        }
+      ]
+    },
+    {
+      name: 'Node 2',
+      level: 1,
+      children: [
+        {
+          name: 'Node 2.1',
+          level: 2,
+          children: []
+        }
+      ]
+    }
+  ];
+</script>
 <h1>Examples</h1>
 
-<Blockquote>TODO</Blockquote>
+Wer're going to use a TreeList to render the following tree using &lt;ul&gt; and &lt;li&gt;:
+<br><br>
+
+<Preview language="typescript" code="{"type Node = { name: string; level: number; children: Node[] };\n\nconst nodes =" + JSON.stringify(nodes, null, 2)}">
+<pre>
+  type Node = &lbrace; name: string; level: number; children: Node[] &rbrace;;
+  const nodes = {JSON.stringify(nodes, null, 2).slice(0, 300)}...
+</pre>
+</Preview>
```

Comment:
I wonder if it might be nicer to add `"svelte-json-tree` as a `devDependency` like  I did for LayerChart [recently](https://github.com/techniq/layerchart/commit/f91b0060892f63b2326edfba3d41a5379be8e203).


https://github.com/techniq/svelte-ux/assets/177476/018bdc16-f9bf-4a11-86fb-84cfda4ab8e7

I would think we would show it inline instead of via a dialog, but just a nice JSON preview.  What do you think?